models/role.mjs

```diff
@@ -1,9 +0,0 @@
-import mongoose from 'mongoose';
-
-const roleSchema = new mongoose.Schema({
-  name: String,
-});
-
-const Role = mongoose.model('Role', roleSchema, 'roles');
-
-export default Role;
```

Comment:
e mai bine sa o ai separata, ca sa o poti importa daca ai nevoie

Reply:
am facut fix in main